code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -95,6 +95,8 @@
 		new /obj/item/ammo_casing/rocket/rpb54(src)
 		new /obj/item/ammo_casing/rocket/rpb54(src)
 		new /obj/item/ammo_casing/rocket/rpb54(src)
+		new /obj/item/ammo_casing/rocket/rpb54(src)
```

Comment:
I think 4 is more than enough, 5 is a bit overkill. Even the RPG bags don't hold as much

Reply:
on campaign RPG guys had 5 rockets in their bag